meta-xt-prod-cockpit-rcar-control/recipes-guests/domd/domd.bbappend

```diff
@@ -23,3 +23,11 @@ do_install_append() {
     echo "[Service]" >> ${D}${systemd_unitdir}/system/domd.service
     echo "ExecStartPre=${libdir}/xen/bin/domd-set-root" >> ${D}${systemd_unitdir}/system/domd.service
 }
+
+do_install_append_scmi() {
```

Comment:
Hi @iusyk, Please move this part to the do_install_append section above so that we do not rely on the same OVERRIDE we currently do not want to use. Let's add it later when we create a working yaml ENABLE_SCMI yaml parameter.

Reply:
Done.